cg_hermes/config/nextflow.py

```diff
@@ -4,12 +4,24 @@
 mandatory by default. However the tags that are available to a particular analysis is mandatory for that analysis.
 """
 
+from cg_hermes.constants.tags import (
+    NextflowTags,
+    QCTags,
+    UsageTags,
+)
+
+
 NEXTFLOW_TAGS = {
     frozenset({"software-versions"}): {
         "is_mandatory": True,
         "bundle_id": True,
-        "tags": ["software-versions"],
-        "used_by": ["cg", "long-term-storage"],
+        "tags": [NextflowTags.SOFTWARE_VERSIONS],
+        "used_by": [UsageTags.CG, UsageTags.LONG_TERM_STORAGE],
+    },
+    frozenset({"qc-metrics"}): {
+        "is_mandatory": True,
+        "tags": [QCTags.QC_METRICS],
+        "used_by": [UsageTags.CG, UsageTags.LONG_TERM_STORAGE],
```

Comment:
why adding this here? this is not a nextflow tag and this is the metrics deliverable files that is common to all workflows . If included here will also be repeated for rnafusion, tomte and taxprofiler which I'm not sure if will be a problem or not

Reply:
I removed it from the individual nextflow pipelines so there is no repetition. 
I agree that it is not a nextflow specific tag, but it is commonly used by all nextflow pipelines. It avoids repetition without having to homogenise all the pipelines (slightly different usage tags) before the tags are refactored.

Reply:
I like it! Less code duplication 👍  probably we'll end up moving these common tags somewhere upstream in the future so it will be removed from nextflow, but for now it's a good place